tests/plugins/tableselection/manual/draganddropimage.html

```diff
@@ -0,0 +1,42 @@
+<textarea id="editor">
+	<table border="1" cellpadding="1" cellspacing="1" style="width:500px">
+		<tbody>
+			<tr>
+				<td><img alt="" src="https://via.placeholder.com/100" />&nbsp;</td>
```

Comment:
What is the purpose of this `&nbsp`? Because without it - when I d&d image - it is pasted with additional borders 🤔 

If you leave it, then it is possible to d&d image inside the same cell, but after `&nbsp`. Then when you d&d image to another cell it is copied with additional borders. It looks like it is copied with the entire `table` construction inside the `td` element.

Reply:
However, it occurs only in chrome... 🤔

Reply:
As you said, without `&nbsp;` d&d is breaking. I created a separate [issue](https://github.com/ckeditor/ckeditor4/issues/4952) for this case. This problem occurs when `tableselection` and `clipboard` plugins are enabled.